src/main/java/org/jabref/gui/Base.css

```diff
@@ -818,7 +818,7 @@
 
 .separator:horizontal .line {
     -fx-border-color: -jr-separator;
-    -fx-border-width: 0;
+    -fx-border-width: 0.5;
```

Comment:
I think there is no need to have visible separators. Just use normal Labels for these section headings and increase the font-size / make them bold. (I think I have done something similar lately, but cannot remember where - there might be a "heading" class defined in the base.css)